chapter02/sxg/src/main/java/com/github/sebin/chapter02/User.java

```diff
@@ -0,0 +1,36 @@
+package com.github.sebin.chapter02;
+
+public class User {
+	private int customerID;
+	private String username;
+	private String password;
+	private String fullname;
```

Comment:
there is no setters, so these parameters are `final`, back to the old topic, are you sure customer's name will never change at all? :smile: